ksy_schema.json

```diff
@@ -296,8 +304,8 @@
         "types": { "$ref": "#/definitions/TypesSpec" },
         "enums": { "$ref": "#/definitions/EnumsSpec" },
         "instances": { "$ref": "#/definitions/InstancesSpec" },
-        "doc": { "type": "string" },
-        "doc-ref": { "type": "string" }
+        "doc": { "$ref": "#/definitions/Attribute/properties/doc" },
+        "doc-ref": { "$ref": "#/definitions/Attribute/properties/doc-ref" }
```

Comment:
Minor thing: the convention in the rest of the spec seems to be that all `"$ref"`s point to a top-level definition under `"definitions"`. For example, a few lines above the spec uses `"types": { "$ref": "#/definitions/TypesSpec" }` and not `"types": { "$ref": "#/properties/types" }`.

Maybe we should put the types of `doc` and `doc-ref` into top-level `"Doc"` and `"DocRef"` definitions and reference those from all properties, rather than making the properties under `"Attribute"` the main definitions that are referenced from elsewhere.